src/kprofile.cpp

```diff
@@ -25,20 +25,26 @@ void KProfile::startDownload(QWebEngineDownloadRequest* download)
 #endif
 {
     auto app = KiwixApp::instance();
-    QString defaultFileName = download->url().fileName();
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+    QString defaultFileName = QUrl(download->path()).fileName();
+#else
+    QString defaultFileName = download->downloadFileName();
+#endif
```

Comment:
This better be done in the previous (or separate) commit

Reply:
Done